types/vue-secure-captcha.d.ts

```
@@ -0,0 +1,5 @@
+declare module 'vue-secure-captcha' {
+  import { DefineComponent } from 'vue';
+  const SecureCaptcha: DefineComponent<{}, {}, any>;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)